src/dso_api/dynamic_api/serializers.py

```diff
@@ -978,15 +978,13 @@ def _through_serializer_factory(  # noqa: C901
             "title", serializers.CharField(source=title_field, read_only=True)
         )
 
-    # Always include the primary identifier of the targeted relationship,
-    id_seq = first(target_table_schema.identifier)  # e.g.: "identificatie"
-    serializer_part.add_field_name(id_seq, source=f"{target_fk_name}_{id_seq}")
-
     # See if the table has historical data
     temporal: Optional[Temporal] = target_table_schema.temporal
     if temporal is not None:
         # Include the temporal identifier of the targeted relationship,
```

Comment:
Moet de commentaar regel niet ook mee verhuisd worden?

Reply:
nope, die was alleen van toepassing op de oude code. Nu voegen we de pk alleen toe als de target temporal is